docs/installation/upgrading.rst

```diff
@@ -1,31 +1,97 @@
 .. _upgrading-ksql:
 
-Upgrading KSQL
-==============
+Upgrading ksqlDB
+================
 
-Upgrade one KSQL server at a time (i.e. rolling restart). The remaining KSQL servers should have sufficient spare
-capacity to take over temporarily for unavailable, restarting servers.
+Upgrade one server at a time in a "rolling restart". The remaining servers
+should have sufficient spare capacity to take over temporarily for unavailable,
+restarting servers.
+
+For general guidance on upgrading, see
+[Upgrade ksqlDB](https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/).
+
+Upgrading to ksqlDB 5.5 from KSQL 5.4
+-------------------------------------
+
+.. important::
+
+   The upgrade from KSQL 5.4 to ksqlDB 5.5 is not a rolling restart. You must
+   shut down all KSQL instances and then start up all ksqlDB instances, so there
+   will be downtime.
+
+#. Prepare a list of SQL statements for migration. One way is to dump the command
+   topic.
+
+   ```bash
+   ./bin/kafka-console-consumer --bootstrap-server localhost:9092 --topic _confluent-ksql-default__command_topic --from-beginning | jq ".statement" 
```

Comment:
I'm not sure about using this command. I might have a mix of CREATE & DROP statements that end up on dropped streams, The user will need to walk through that list and figure out which streams and tables do not exist anymore.

Also, what's the difference between this step and the next one that says `List your application's streams ...`?  Isn't the same?

Reply:
#4971 just as an FYI

Reply:
See my other comment :)

Reply:
I reworked the instructions for getting the SQL statements to be more like https://docs.ksqldb.io/en/latest/operate-and-deploy/installation/upgrading/#how-to-upgrade.